app/sprinkles/core/config/default.php

```diff
@@ -232,9 +232,9 @@
     'mail'    => [
         'mailer'          => 'smtp', // Set to one of 'smtp', 'mail', 'qmail', 'sendmail'
         'host'            => getenv('SMTP_HOST') ?: null,
-        'port'            => 587,
-        'auth'            => true,
-        'secure'          => 'tls', // Enable TLS encryption. Set to `tls`, `ssl` or `false` (to disabled)
+        'port'            => getenv('SMTP_PORT') ?: null,
+        'auth'            => getenv('SMTP_AUTH') ?: null,
+        'secure'          => getenv('SMTP_SECURE') ?: null, // Enable TLS encryption. Set to `tls`, `ssl` or `false` (to disabled)
```

Comment:
Hum... This would create a breaking change for people without those settings already in their env, which push this change to UF 4.5. I wonder if there's a way to keep the default here, and overwrite in the env?